applications/ChimeraApplication/python_scripts/navier_stokes_solver_vmsmonolithic_chimera.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.ChimeraApplication as KratosChimera
+# Import applications
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# Import base class file
+from navier_stokes_solver_vmsmonolithic import NavierStokesSolverMonolithic
```

Comment:
```suggestion
from KratosMultiphysics.FluidDynamicsApplication.navier_stokes_solver_vmsmonolithic import NavierStokesSolverMonolithic
```
otherwise will stop working soon

Reply:
Done I changed all of such occurrences !